src/app/views/sessions/signin/signin.component.ts

```diff
@@ -342,6 +343,15 @@ export class SigninComponent implements OnInit, OnDestroy {
   }
 
   gotoTC() {
-    window.open(this.tc_ip);
+    this.dialogService.generalDialog({
+      title: helptext.tcDialog.title,
+      message: helptext.tcDialog.message,
+      is_html: true,
+      confirmBtnMsg: helptext.tcDialog.confirmBtnMsg,
+    }).subscribe(res => {
+      if (res) {
+        window.open(this.tc_ip);
```

Comment:
where does TC IP come from? We are probably going to have two attributes, one for URL and one for IP